docs/src/charmil_cli_crud.md

```diff
@@ -0,0 +1,70 @@
+# Charmil CRUD Generator
+
+- With the help of the `charmil crud` command, developers can eliminate a lot of boilerplate in CLIs containing multiple services that perform standard CRUD operations.
+
+- Using a set of pre-defined templates, this command generates CRUD command packages in the directory specified by the `crudpath` flag as well as its corresponding language file in the directory specified by the `localepath` flag.
+
+- These generated files can then be modified by developers to fit their own needs.
+
+## Usage:
+
+```bash
+charmil crud [flags]
+```
+
+## Flags:
+
+```
+  -c, --crudpath string     path where CRUD files need to be generated (default ".")
+  -h, --help                help for crud
+  -l, --localepath string   path where the language file needs to be generated (default ".")
+  -p, --plural string       name in plural form (REQUIRED)
+  -s, --singular string     name in singular form (REQUIRED)
+```
+
+## Steps to use:
+
+- Let's say you need to generate CRUD commands for managing your Kafka instances, the following command can be used for the same:
+
+  ```bash
+  $ charmil crud --singular=kafka --plural=kafkas --crudpath="./kafka" --localepath="./cmd/locales/en"
+  ```
+
+- On running the command mentioned above, the required files will be generated in your project in the following structure:
+
+  ```code
+  📦Your CLI
+   ┣ 📂cmd
+   ┃ ┗ 📂locales
+   ┃   ┗ 📂en
+   ┃     ┗ 📜crud.en.yaml
+   ┗ 📂kafka
+     ┣ 📂create
+     ┃ ┣ 📜create.go
+     ┃ ┗ 📜run.go
+     ┣ 📂delete
+     ┃ ┣ 📜delete.go
+     ┃ ┗ 📜run.go
+     ┣ 📂describe
+     ┃ ┣ 📜describe.go
+     ┃ ┗ 📜run.go
+     ┣ 📂list
+     ┃ ┣ 📜list.go
+     ┃ ┗ 📜run.go
+     ┣ 📂use
+     ┃ ┣ 📜use.go
+     ┃ ┗ 📜run.go
+     ┗ 📜root.go
```

Comment:
that can be kafka.go

Reply:
Resolved via commit: https://github.com/aerogear/charmil/pull/198/commits/8af52def6450ec6265639f416f1dafc87e477207